crates/tests/src/e2e/ledger_tests.rs

```diff
@@ -2808,3 +2820,407 @@ fn test_genesis_manipulation() -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn test_mainnet_phases() -> Result<()> {
+    // Use 1 PGF steward
+    let steward = defaults::albert_address();
```

Comment:
Mainnet will have no stewards at genesis, so we should do the same. Perhaps in this test we can elect one in Phase 1 or at some later point?

Reply:
applied this suggestion in last version